tests/Feature/OrganizationInvitationTest.php

```diff
@@ -163,4 +163,59 @@ public function testAcceptInvitationUnauthorizedIfEmailMismatch(): void
         $response->assertStatus(Response::HTTP_FORBIDDEN);
         $response->assertJson(["message" => __("organization.invitation_unauthorized")]);
     }
+
+    public function testInvitationThrottleWorks(): void
+    {
+        $this->actingAs($this->owner);
+
+        $response1 = $this->postJson("/api/organizations/{$this->organization->id}/invite", [
+            "email" => $this->invitee->email,
+        ]);
+        $response1->assertOk();
```

Comment:
We don't really need to keep response in variable. You can do something like this:
```suggestion
        $this->postJson("/api/organizations/{$this->organization->id}/invite", [
            "email" => $this->invitee->email,
        ])->assertOk();
```